### PR TITLE
Bump stable channel with alpha; alpha channel with latest

### PR DESCRIPTION
**Pull request by justinsb**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2425)

### CODE REVIEW

File: channels/stable [hunk 1]
```diff
@@ -7,13 +7,15 @@ spec:
...
+    - providerID: gce
+      name: "cos-cloud/cos-stable-56-9000-84-2"
   cluster:
     kubernetesVersion: v1.4.8
     networking:
       kubenet: {}
   kubernetesVersions:
   - range: ">=1.5.0"
-    recommendedVersion: 1.5.2
+    recommendedVersion: 1.5.4
```
Comment (chrislovecnm) [inline channels/stable]: 1.5.6?

Comment (justinsb) [inline channels/stable]: We haven't tested it in the alpha channel.  alpha -> stable, latest -> alpha

Comment (chrislovecnm) [inline channels/stable]: Raises had ... tested it.

Comment (chrislovecnm) [inline channels/stable]: I have tested it ... but no biggy. Merging we can do another PR.
